test/tools/libtesteth/Options.cpp

```diff
@@ -210,7 +209,7 @@ Options::Options(int argc, char** argv)
 		else if (arg == "--seed")
 		{
 			throwIfNoArgumentFollows();
-			randomTestSeed = static_cast<uint64_t>(toInt(argv[++i]));
+			randomTestSeed = static_cast<uint64_t>(min<u256>(UINT64_MAX, toInt(argv[++i])));
```

Comment:
What's the difference? You sill hide the invalid value error.

Reply:
What is hidden? Invalid value errors about negative `int`s?